src/Pages/ProductList/ProductListMain/MainHeader/FilterToggle/FilterModal/RefundPolicyModal/RefundPolicyModal.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import styled from "styled-components";
+import mixin from "../../../../../../../Styles/mixin";
+import ProductListSvg from "../../../../../ProductListSvg/ProductListSvg";
+
+export default function RefundPolicyModal({
+  onChecked,
+  onFocused,
+  isChecked,
+  isFocused,
+}) {
+  return (
+    <HeadWrapper>
+      <div>유연한 환불 정책을 제공하는 숙소만 검색 결과에 표시</div>
+      <HeadBtnBox onClick={onChecked} isFocused={isFocused}>
+        <BtnBack isChecked={isChecked} onFocus={onFocused} onBlur={onFocused}>
```

Comment:
```suggestion
        <BtnBack isChecked={isChecked} onFocused={onFocused} onBlur={onFocused}>
```
변수명이 갑자기 달라질 이유는 없어보이네요!

Reply:
아 저거 onFocus 이벤트 입니당

Reply:
ㅎㅎ 실수